src/content/docs/references/docs/optionals.md

```diff
@@ -5,187 +5,272 @@ sidebar:
     order: 115
 ---
 
-Optionals in C3 work differently from other languages:
+## Optionals handle potential failure 
 
-1. It is similar to a "Result" type in capabilities.
-2. It is not quite a type: it can be used for variable and return values - but not for parameters or struct member types.
-3. It "cascades" the optional-ness of an expression, somewhat reminiscent of the "flatmap" operation.
-4. A function called with an Optional is only invoked if it has an actual value.
- 
-## What is an Optional?
+A common example using optionals is for a function that can return a value or it can fail. We call the reason for the failure a `fault` and the successful outcome a `result`. 
```

Comment:
Can we further stress that it's just that a function might not be able to provide a value rather than that something needs to be an error?